### ISSUE TITLE
negative training loss

### ISSUE THREAD

**Issue by czha5168** [Jia-Research-Lab/PointGroup#20]
Hi Liang, Thanks for sharing your well-written codes. Just a kindly reminder that when exploring your repo, I noticed some negative training loss appeared after ~50 epochs. I guess it might be caused by some typo, which should not affect the algorithm correctness (as their validation loss looks all good). Looking forward to your comments, and wish all the best.

_Some log info:_
epoch: 58/384, train loss: 0.0207
Saving exp/scannetv2/pointgroup/pointgroup_run2_scannet/pointgroup_run2_scannet-000000058.pth
epoch: 59/384, train loss: 0.0062
Saving exp/scannetv2/pointgroup/pointgroup_run2_scannet/pointgroup_run2_scannet-000000059.pth
epoch: 60/384, train loss: -0.0002
Saving exp/scannetv2/pointgroup/pointgroup_run2_scannet/pointgroup_run2_scannet-000000060.pth
epoch: 61/384, train loss: -0.0171
Saving exp/scannetv2/pointgroup/pointgroup_run2_scannet/pointgroup_run2_scannet-000000061.pth
epoch: 62/384, train loss: -0.0477
Saving exp/scannetv2/pointgroup/pointgroup_run2_scannet/pointgroup_run2_scannet-000000062.pth
epoch: 63/384, train loss: -0.0609
Saving exp/scannetv2/pointgroup/pointgroup_run2_scannet/pointgroup_run2_scannet-000000063.pth
epoch: 64/384, train loss: -0.0646
Saving exp/scannetv2/pointgroup/pointgroup_run2_scannet/pointgroup_run2_scannet-000000064.pth
Start Evaluation: epoch: 64/384, val loss: 2.0182

**Comment by llijiang** [comment]
Hi,
The offset direction loss could have a negative value.  It is to be expected to have a negative training loss.

**Comment by czha5168** [comment]
Hi Liang, Thanks for your quick reply and comments!
